CONTRIBUTING.md

```diff
@@ -12,7 +12,33 @@ Please consult the [README](./README.md) file at the root of this repository.
 
 ## How to Contribute
 
-Please follow the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website.
+Please read the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website. It contains some general instructions that should be followed when contributing to any of the Creative Commons open-source repositories.
```

Comment:
I think it would be worth mentioning here that `CODEBASE.md` is a good place to find high-level information.